src/scss/_action.scss

```diff
@@ -34,58 +34,44 @@
     justify-content: center;
 
     //content unique to this instance of c-action__link
-    background-color: $orange;
+    background-color: darken($green-brand, 2%);
```

Comment:
Thanks! And if you have any design or content suggestions feel free to let me know! Does anything appear "off" to you?